backend/infrahub/core/schema/schema_branch.py

```diff
@@ -1522,18 +1522,22 @@ def _get_hierarchy_child_rel(self, peer: str, hierarchical: str, read_only: bool
             read_only=read_only,
         )
 
-    def _get_hierarchy_parent_rel(self, peer: str, hierarchical: str, read_only: bool) -> RelationshipSchema:
+    def _get_hierarchy_parent_rel(
+        self, peer: str, hierarchical: str, read_only: bool, optional: str
```

Comment:
The `hierarchical` we're sending in is of type `str | None`, also `RelationshipSchema` expects `optional` to be a boolean.

Reply:
Slow brain + mypy ignore = this